src/app/graphicsUtils.js

```diff
@@ -0,0 +1,31 @@
+define([
+  'esri/graphicsUtils'
+], function (
+  graphicsUtils
+) {
+    return {
+        unionGraphicsIntoExtent: function (graphics) {
```

Comment:
[`graphicsExtent`](https://developers.arcgis.com/javascript/jsapi/esri.graphicsutils-amd.html#graphicsextent) takes an array of multiple extents also. `unionGraphicsIntoExtent` seems to be a duplicate of ESRI's `graphicsExtent`.